modules/microshift-storage-classes.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-classes_{context}"]
+== Storage classes
+
+Storage classes provide the workload layer interface for selecting a device class. The following storage class parameters are supported in {product-title}:
+
+* `csi.storage.k8s.io/fstype`, which selects the filesystem types. `xfs` or `ext4` filesystem type sare supported
+* `topolvm.io/device-class` is the name of the device class. If a device class is not provided, the default device class is assumed.
+
+Multiple storage classes can refer to the same device class. You can provide varying sets of parameters for the same backing device class, such as `xfs` and `ext4` variants.
+
+.{product-title} defines the following as the default storage class:
```

Comment:
@copejon, does the user need to provide an additional StorageClass for snapshotting? If so, we need to provide an example of what that looks like.

Reply:
This is a kube API resource. It exists in the database, but not in a separate configuration file. We want to document the defaults in this format because it's the same format a user would need to use to add their own storage class.